### PR TITLE
Adding the base structure to write cops

### PR DESCRIPTION
**Pull request by juanmanuelramallo**
- Added the cop folder to write cops
- Updated readme with a link to Rubocop's documentation on how to write custom cops
- Using our default config file in the project

### CODE REVIEW

File: lib/rubocop/cop/rootstrap.rb [hunk 1]
```diff
@@ -0,0 +1 @@
+# require "rubocop/cop/rootstrap/custom_cop"
```
Comment (fedeagripa) [inline lib/rubocop/cop/rootstrap.rb]: It is right this to be commented ?

File: rubocop-rootstrap.gemspec [hunk 2]
```diff
@@ -15,8 +13,12 @@ Gem::Specification.new do |spec|
 
   spec.files = Dir['LICENSE.txt', 'README.md', 'lib/**/*', 'config/*.yml']
 
-  # Rubocop version is constrained to these versions to exclude breaking changes from previous versions
+  #
+  # Rubocop version is constrained to these versions to exclude
+  # breaking changes from previous versions.
```
Comment (fedeagripa) [inline rubocop-rootstrap.gemspec:18]: Weird comment # usage here

File: lib/rubocop/cop/rootstrap.rb (hunk 1 above)
Comment (juanmanuelramallo) [reply lib/rubocop/cop/rootstrap.rb]: I can remove it, it's just an example on how to require cops.

File: rubocop-rootstrap.gemspec (hunk 2 above)
Comment (juanmanuelramallo) [reply rubocop-rootstrap.gemspec:18]: Suggestions?

File: lib/rubocop/cop/rootstrap.rb (hunk 1 above)
Comment (fedeagripa) [reply lib/rubocop/cop/rootstrap.rb]: as you are documenting in the readme I think is enough there

File: rubocop-rootstrap.gemspec (hunk 2 above)
Comment (fedeagripa) [reply rubocop-rootstrap.gemspec:18]: Nevermind, It just looked weird with some extra jumps in the mobile app, now that I check it in web is perfect

File: lib/rubocop/cop/rootstrap.rb (hunk 1 above)
Comment (juanmanuelramallo) [reply lib/rubocop/cop/rootstrap.rb]: Yep, makes sense

Comment (juanmanuelramallo) [reply lib/rubocop/cop/rootstrap.rb]: Done

File: rubocop-rootstrap.gemspec [hunk 3]
```diff
@@ -1,10 +1,8 @@
 # frozen_string_literal: true
 
-require_relative 'lib/rubocop/rootstrap/version'
-
 Gem::Specification.new do |spec|
   spec.name          = 'rubocop-rootstrap'
-  spec.version       = Rubocop::Rootstrap::VERSION
```
Comment (santib) [inline rubocop-rootstrap.gemspec:7]: 👀

**Review by santib — APPROVED** [review]

**Review by fedeagripa — APPROVED** [review]

**Review by juan-apa — APPROVED** [review]

**Pull request merged**